sbi/utils/sbiutils.py

```diff
@@ -216,3 +216,34 @@ def warn_on_invalid_x(num_nans: int, num_infs: int, exclude_invalid_x: bool) ->
                 f"Found {num_nans} NaN simulations and {num_infs} Inf simulations. "
                 "Training might fail. Consider setting `exclude_invalid_x=True`."
             )
+
+
+def get_data_after_round(
```

Comment:
Ah, now I get it. Maybe we find a more descriptive name here, what about changing the signature to 

`get_data_since_round(starting_idx, data, data_round_indices)`

?